packages/terra-menu/src/MenuItem.jsx

```diff
@@ -299,11 +299,17 @@ class MenuItem extends React.Component {
       <>
         { MenuUtils.isMac() && <VisuallyHiddenText text={intl.formatMessage({ id: 'Terra.menu.index' }, { index: index + 1, totalItems })} /> }
         { MenuUtils.isMac() && (isGroupItem || toggleable) && <VisuallyHiddenText text={markAsToggled ? intl.formatMessage({ id: 'Terra.menu.selected' }) : intl.formatMessage({ id: 'Terra.menu.unselected' })} /> }
+        {/* Adds context to Use the up and down arrows to navigate the options */}
+        { this.itemNode && this.itemNode.parentNode.getAttribute('data-submenu') === 'true' && index === 0 && totalItems !== 1 && !MenuUtils.isMac()
+          && <VisuallyHiddenText text={intl.formatMessage({ id: 'Terra.menu.navigateMenuItem' })} /> }
         {/* Adds context for item with submenu-items */}
         { subMenuItems.length > 0 && <VisuallyHiddenText text={intl.formatMessage({ id: 'Terra.menu.itemWithSubmenu' })} /> }
         {/* Adds context for navigating back to parent menu from submenu */}
         { this.itemNode && this.itemNode.parentNode.getAttribute('data-submenu') === 'true' && index === 0
             && <VisuallyHiddenText text={intl.formatMessage({ id: 'Terra.menu.exitSubmenu' })} /> }
+        {/* Adds context for selected and unselected state */}
+        { !MenuUtils.isMac() && toggleable && !isGroupItem && <VisuallyHiddenText aria-live="polite" text={markAsToggled ? intl.formatMessage({ id: 'Terra.menu.selected' }) : intl.formatMessage({ id: 'Terra.menu.unselected' })} /> }
+        { !MenuUtils.isMac() && isGroupItem && <VisuallyHiddenText aria-live={markAsToggled ? 'polite' : undefined} text={markAsToggled ? intl.formatMessage({ id: 'Terra.menu.selected' }) : intl.formatMessage({ id: 'Terra.menu.unselected' })} />}
```

Comment:
we can have single visually hidden text component for both selected and unselected state.

Reply:
We were using single visually hidden text for selected and unselected state. here aria-live attribute is varying for menuitemcheckbox and menuiteradio, that's why we have added two VisuallyHiddenText.

Reply:
Updated single visually hidden text component for both selected and unselected state - [339656d](https://github.com/cerner/terra-framework/pull/1718/commits/339656d46490613a1c21e937904e476e954f6a7a)